RELEASE.md

```diff
@@ -41,58 +41,68 @@ with `make upgrade-npm-deps`
 create `release/<version>` for patch or release candidate releases.
 
-- Do this in a proper PR pointing to the release branch as this gives others the opportunity to chime in on the release
-  in general and on the addition to the changelog in particular.
+#### Create a PR with the changes
 
+- Create a branch with the changes for the release (e.g. `yourname/release-v<major>.<minor>.<patch>`). 
 - Update the file `VERSION` with the new version to be created.
-
-- Update the file `CHANGELOG.md` :
-
-```bash
-make generate-changelog
-```
-
-Note: it will generate a first changelog version based on the git history. You should review what has been generated.
-Error can happen ;)
-
-- Entries in the `CHANGELOG.md` are meant to be in this order:
-
-* `[FEATURE]`
-* `[ENHANCEMENT]`
-* `[BUGFIX]`
-* `[BREAKINGCHANGE]`
-
-As we have many libraries we publish, it's better if you also put a clear indication about what library is affected by
+- Generate `CHANGELOG.md` updates based on git history:
+  ```bash
+  make generate-changelog
+  ```
+- Review the generated `CHANGELOG.md` for valid output. Things to check include:
+  - Entries in the `CHANGELOG.md` are meant to be in this order:
+    * `[FEATURE]`
+    * `[ENHANCEMENT]`
+    * `[BUGFIX]`
+    * `[BREAKINGCHANGE]`
+  - Entries that map to a pull request should include a pull request number.
+  - As we have many libraries we publish, it's better if you also put a clear indication about what library is affected by
 these changes.
-
-- Update the different `package.json` with the corresponding version:
-
-```bash
-make bump-version
-```
+  - Consumers understand how to handle breaking changes either through the messaging in the changelog or through the linked pull requests.
+- Update the `package.json` files for all packages with the corresponding version:
+  ```bash
+  make bump-version
+  ```
+- Push the branch to github and create a pull request with the release branch as the base. This gives others the opportunity to chime in on the release,
+  in general, and on the addition to the changelog, in particular. 
+  - It's also helpful to drop a link to the release PR in [#perses-dev](https://matrix.to/#/#perses-dev:matrix.org) to get extra visibility.
+- Address any necessary feedback.
+- Once the pull request is approved, merge the it into the release branch.
 
 ### 2. Draft the new release
 
-Tag the new release via the following commands:
-
-```bash
-git checkout release/v<version_to_be_replaced>
-make tag
-git push origin v<version_to_be_replaced>
-```
+- Pull down the latest updates to the release branch on your local machine to ensure you have the updates from the previous step.
+- Tag the new release via the following commands:
+  ```bash
+  git checkout release/v<version_to_be_replaced>
+  make tag
+  git push origin v<version_to_be_replaced>
+  ```
 
 Signing a tag with a GPG key is appreciated, but in case you can't add a GPG key to your Github account using the
```

Comment:
@cndonovan I recall you having issues with the GPG key bit of this. Is there anything you learned that's worth including here for future folks?

Reply:
I had an issue when I first got my M1 Mac that I posted about [here](https://matrix.to/#/!XNmgYIAndZOkEvQAbb:matrix.org/$1RHQzJMV44vquFgoNTPaKm8ozFcOhR-XrpWEAXkAZAk?via=matrix.org&via=herail.net), at the time, this SO article helped me: https://stackoverflow.com/questions/39494631/gpg-failed-to-sign-the-data-fatal-failed-to-write-commit-object-git-2-10-0/40066889#40066889

I thought about adding a note about that in RELEASE.md back then but it did feel very specific to my setup and I was hoping the issue when get fixed eventually, I think Christine might have run into it too though, curious did you have any similar issues with your M1?

Reply:
i remember running into this issue as well

Reply:
I added a note about that issue and a link to the SO article.